Makefile

```diff
@@ -134,6 +135,36 @@ build-tf2-gpu:
 		-t $(NGC_REGISTRY)/$(GPU_TF2_ENVIRONMENT_NAME)-$(VERSION) \
 		.
 
+.PHONY: build-deepspeed-gpu
+build-deepspeed-gpu:
+	# Same base image is used for deepspeed as that for the tf2-gpu image.
+	docker build -f Dockerfile-base-gpu \
+		--build-arg BASE_IMAGE="nvidia/cuda:11.1-cudnn8-devel-ubuntu18.04" \
```

Comment:
Can we use nvidia/cuda:11.1-cudnn8-devel-ubuntu20.04 instead? We're currently pretty well standardized on that version.

Reply:
i took the same base image as our tf2-gpu image; it also looks like the other images in the Makefile use ubuntu18.04 unless something changed recently.

Reply:
Huh - I thought we had standardized everything. Okay, nevermind. We can update them all at once later - and the next Ubuntu LTS will be out soon anyway.